hw/formal/tools/jaspergold/parse-formal-report.py

```diff
@@ -73,25 +73,25 @@ def parse_fpv_summary(str_buffer):
                         ("covered", r"^\[\d+\].*covered.*"),
                         ("undetermined", r"^\[\d+\].*undetermined.*"),
                         ("unreachable", r"^\[\d+\].*unreachable.*")]
-    fpv_summary = extract_messages_count(str_buffer, message_patterns)
+    f_summary = extract_messages_count(str_buffer, message_patterns)
```

Comment:
nit: Since the script already very much pertains to 'formal' I think you can omit 'formal' from the variable/function names.

Reply:
Got it, thanks Sri. i will update that.